subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -67,23 +68,33 @@ public void buildFinished(BuildResult result) {
 
     public void execute(Throwable failure) {
         if (failure instanceof MultipleBuildFailures) {
-            renderMultipleBuildExceptions((MultipleBuildFailures) failure);
+            List<Throwable> flattenedFailures = new ArrayList<Throwable>();
+            flattenMultipleBuildFailures((MultipleBuildFailures) failure, flattenedFailures);
+            renderMultipleBuildExceptions(flattenedFailures);
```

Comment:
:x: This logic warrants a test case in `BuildExceptionReporterTest`

Reply:
done